package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "eslint src test",
     "fix": "eslint src test --fix",
     "test": "mocha -t 20000",
-    "ci": "npm run lint && cross-env DEBUG_CALLS=true npm run test",
+    "ci": "npm run lint && cross-env npm run test",
```

Comment:
Whoops, nevermind then. I don't have a dog in the hunt, just was concerned about PR scope.